modules/security/src/main/java/io/oasp/module/security/common/api/accesscontrol/AbstractAccessControl.java

```diff
@@ -84,7 +84,7 @@ public boolean equals(Object obj) {
     if (getClass() != obj.getClass()) {
       return false;
     }
-    AccessControl other = (AccessControl) obj;
+    AbstractAccessControl other = (AbstractAccessControl) obj;
```

Comment:
Hi @amarinso , These changes are because of renaming the class to AbstractAccessControl from AccessControl. We can keep the name as it is .i.e, AccessControl . Pls let me know.   

Thanks,
@kiran-vadla